src/lib_ccaligner/params.cpp

```diff
@@ -495,31 +504,14 @@ void Params::inputParams(int argc, char *argv[])
 void Params::validateParams()
 {
     if(audioFileName.empty() && !readStream)
+    {
         FATAL(EXIT_INVALID_PARAMETERS, "Audio file name is empty!");
+    }
 
     if(subtitleFileName.empty())
+    {
         FATAL(EXIT_INVALID_PARAMETERS, "Subtitle file name is empty!");
-
-    if(modelPath.empty())
-        LOG("Using default Model Path.");
-
-    if(dictPath.empty())
-        LOG("Using default Dictionary Path.");
-
-    if(lmPath.empty())
-        LOG("Using default LM Path.");
-
-    if(logPath.empty())
-        LOG("Using default Log Path.");
-
-    if(fsgPath.empty())
-        LOG("Using default FSG Path.");
-
-    if(phoneticlmPath.empty())
-        LOG("Using default Phonetic LM Path.");
-
-    if(phonemeLogPath.empty())
-        LOG("Using default Phoneme Log Path.");
+    }
```

Comment:
Personally, I prefer not to add braces when it only has only one statement because it will make it harder to read, and less "elegant".

Reply:
@harrynull Yeah, me too. But everywhere else there are braces, even for single statements, and so for consistency, I'll need to add them.

Reply:
@Rakete1111 Not really "everywhere else". I think it's half and half. You can find a lot braces are leaved out if you look at generate_approx_timestamp.cpp. It would be much better if there is a official written guideline.

Reply:
Guys, the reason I am reluctant to adapt a very strict styling guidelines is that most of the people working on it 
are junior students, who might now be very experienced. I just want them to have a smooth GCI. Having said that it doesn't mean we shouldn't have any. 

If you guys want - please create an issue on Github issue tracker regarding a single styling guideline, and upon discussion come-up with a single guide. Just keep in mind 

- should be easy to comprehend 
- no unnecessary restrictions
- should help developer focus more on work, than worrying about formatting
- of course, result in simple, and beautiful code :)